domains.txt

```diff
@@ -260,11 +295,13 @@ giostechnology.com
 glglobal.co.uk
 global-m.co.uk
 global-resourcing.com
+globalsearchgroup.co.uk
 glocomms.com
 glotel.co.uk
 goldgroup.co.uk
 gqrgm.com
 gravitasrecruitmentgroup.com
+grb.uk.com
```

Comment:
Now in our 20th year, Graduate Recruitment Bureau (GRB) is the UK's highest review-rated graduate recruitment consultancy. Every day our teams of sector-specific experts get contacted by major graduate recruiters, SMEs and start-ups who are looking for high calibre university students and graduates for their full-time graduate jobs, graduates schemes, placement years and internships.